src/solver/batchnorm/backward_per_activation_fused.cpp

```diff
@@ -233,6 +233,11 @@ ConvSolution BnBwdTrgActivationFused::GetSolution(const FusionContext& context,
     return result;
 }
 
+float BnBwdTrgActivationFused::GetWti(const FusionContext&, const FusionDescription&) const
+{
+    return -2;
+}
```

Comment:
This function can be removed, because it returns the default value

Reply:
Makes sense. Will remove default value ones.